pkg/detectors/azurestorage/azurestorage_test.go

```diff
@@ -112,6 +119,25 @@ func TestAzurestorage_FromChunk(t *testing.T) {
 				data:   []byte(fmt.Sprintf("You can find a azurestorage secret %s within", secret)),
 				verify: true,
 			},
+			want: func() []detectors.Result {
+				r := detectors.Result{
+					DetectorType: detectorspb.DetectorType_AzureStorage,
+					Verified:     false,
+				}
+				r.SetVerificationError(fmt.Errorf("unexpected HTTP response status 404"), secret)
+				return []detectors.Result{r}
+			}(),
+			wantErr:             false,
+			wantVerificationErr: true,
```

Comment:
same question as that other pr: is this flag necessary?

Reply:
nope